daemon/algod/api/algod.oas2.json

```diff
@@ -304,6 +317,10 @@
     "/v2/accounts/{address}/applications/{application-id}": {
       "get": {
         "description": "Given a specific account public key and application ID, this call returns the account's application local state and global state (AppLocalState and AppParams, if either exists). Global state will only be returned if the provided address is the application's creator.",
+        "tags": [
+          "public",
```

Comment:
Thinking out loud. It seems a little unfortunate that we have to split up all endpoints twice for public/private. Maybe there is an oapi-codegen feature we could add, or a different way to manage tokens.

Reply:
I'll look into this--obviously the reason different routes are generated is to provide different authentication handlers. Potentially there is a way to specify that in the generation, but I'm not sure.

Reply:
It looks like they now provide some basic methods for constructing common auth mechanism (https://github.com/deepmap/oapi-codegen/blob/master/pkg/securityprovider/securityprovider.go), but I don't see anything that will let us easily merge these.

Reply:
Thanks for looking. This isn't a blocking issue so I'm going to resolve the thread.